docs/operating-scylla/procedures/backup-restore/backup.rst

```diff
@@ -69,7 +69,7 @@ Incremental Backup
 |  * A snapshot
 |  * All incremental backups and commit logs from the time of the snapshot.
 
-| Note: make sure to delete unnecessary incremental backups. Scylla does not do this automatically.
+| Note: make sure to delete incremental backups files after copying them to external storage. Scylla does not do this automatically.
```

Comment:
the `snapshots` and `backups` subdirectories aren't meant for data retention.
They just keep temporary state for an external tool to copy the sstables out to some external storage - which is used for data retention.
Once the sstables are copied to the backup medium from snapshots/backups they can be removed.
With `snapshots` it's easier, since a subdirectory per snapshot operation is created under snapshots and it can be easily removed as a whole using the proper scylla api.
But we don't have such a thing for `backups` since it is used for "continuous" snapshots and new sstables continue to be linked into it as they are sealed.  The poor user of `backups` needs to manage its contents by some means, like scanning the directory once in a while, copy all the scanned sstables externally, and then delete them.
New sstables may be linked into `backups` concurrently, but that shouldn't be a problem.
They should be picked up on the next round.